composer.json

```diff
@@ -147,7 +147,6 @@
     "require-dev": {
         "ext-fileinfo": "*",
         "bamarni/composer-bin-plugin": "^1.4",
-        "composer/composer": "^1.0 || ^2.0",
```

Comment:
I think we still need this for the manager bundle?

Reply:
Seems the dependency was missing there all the time. 😄

Reply:
Re-added in edcc0a5b4f8684e86995324e5a5ab44bf0f90043.